plugins/win-capture/graphics-hook/gl-capture.c

```diff
@@ -809,6 +815,9 @@ static BOOL WINAPI hook_wgl_swap_buffers(HDC hdc)
 
 static BOOL WINAPI hook_wgl_swap_layer_buffers(HDC hdc, UINT planes)
 {
+	if (!gl_functions_initialized)
+		return FALSE;
```

Comment:
Style:  For consistency with other `return` statements, write "FALSE" in lowercase letters.